adhocracy4/datetimefield/apps.py

```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
```

Comment:
This an app because the template should be over writable? I am not saying we should not do this. But does this approach to add an app for each widget really scale?

I would propose to keep this either as an `adhocracy4.forms` package, or if it has to be `adhcoracy4.forms` could also be an app.

Reply:
That makes sense. Renamed the app.